### DISCUSSION TITLE
Development paused? or discontinued?

### DISCUSSION THREAD

**Discussion by tennox** [gzuidhof/starboard-notebook#145] [General]
Hey there :)

Would love to have an idea what's the situation, as I don't see any activity anywhere since > 1 year :cry: 

Also asking because I love starboard and would love to contribute & develop further :)

**Comment by gzuidhof** [comment]
Hey @tennox,

The main bottleneck is my own time :(. I'm co-running a startup that is unrelated to Starboard, which means this project has been in more of a maintenance mode since. I use it myself from time to time, and so do others too - it still works (and perhaps will forever as long was the web is backwards compatible).

I think it deserves more love than it's getting still..

Here's an invite link
https://discord.gg/t5UnYCsS

**Reply by gotjoshua** [reply]
Hi, can you maybe make an invite that doesn't expire so fast?
